src/components/modals/SurveyModal.tsx

```diff
@@ -187,36 +185,38 @@ export const SurveyModal = (props: { bridge: AirshipBridge<void> }) => {
   )
 }
 
-const SurveyModal2 = (props: { bridge: AirshipBridge<boolean>; category: string }) => {
+const SurveyModal2 = (props: { bridge: AirshipBridge<boolean>; category: keyof InstallSurvey2['surveyOptions2'] }) => {
```

Comment:
Your could use the `SurveyOption2Keys` type instead of `keyof InstallSurvey2['surveyOptions2']`, since it's the same thing.